packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-federated-sign-in/amplify-federated-sign-in.component.ts

```diff
@@ -16,16 +16,17 @@ export class AmplifyFederatedSignInComponent implements OnInit {
   public shouldShowFederatedSignIn = false;
 
   // translated texts
-  public signInAmazonText = translate('Sign In with Amazon');
-  public signInAppleText = translate('Sign In with Apple');
-  public signInFacebookText = translate('Sign In with Facebook');
-  public signInGoogleText = translate('Sign In with Google');
+  public signInAmazonText: String;
+  public signInAppleText: String;
+  public signInFacebookText: String;
+  public signInGoogleText: String;
```

Comment:
Minor nit: we should use `string` instead of `String` here:

```suggestion
  public signInAmazonText: string;
  public signInAppleText: string;
  public signInFacebookText: string;
  public signInGoogleText: string;
```

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

Reply:
Oops yup, I'll fix that.